src/scss/styles-1-large-main.scss

```diff
@@ -147,6 +147,26 @@ $headersize: 1.4rem;
   opacity: 1;
 }
 
+.fa-social-icon {
+  color :#152935;
+}
+
+.fa-stack {
+  border-radius: 50px;
+  box-sizing: border-box;
+  border: 1px solid transparent;
+  transition:.5s;
+  background:#ffffff;
+  opacity: .5;
+  
```

Comment:
whitespace

Reply:
Ok, Wasn't sure they had to be removed, will take them out as well when changing the icon size

Reply:
Checked spacing, obsolete images removed